workbench/patches/3.0.0-beta/Update.cfc

```diff
@@ -197,11 +196,15 @@ component implements="contentbox.model.updates.IUpdate"{
 
 	private function updatePermissions(){
 		// Update Old Permissions
-		var perm = permissionService.findWhere( { name="LAYOUT_ADMIN" } );
+		var perm = permissionService.findWhere( { name="THEME_ADMIN" } );
 		if( !isNull( perm ) ){
 			perm.setName( "THEME_ADMIN" );
 			perm.setDescription( "Ability to manage themes, default is view only" );
 			permissionService.save( entity=perm, transactional=false );
+		}else{
```

Comment:
I don't get this part @Tropicalista  It seems to re-insert if the theme admin perm is not there, else try to look for the other on.  Everything else looks great except this portion that doesn't make sense to me.